generated/gitops-template/jenkins/Jenkinsfile

```diff
@@ -19,6 +19,7 @@ pipeline {
         TRUSTIFICATION_OIDC_CLIENT_ID = credentials('TRUSTIFICATION_OIDC_CLIENT_ID')
         TRUSTIFICATION_OIDC_CLIENT_SECRET = credentials('TRUSTIFICATION_OIDC_CLIENT_SECRET')
         TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION = credentials('TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION')
+        QUAY_IO_CREDS = credentials('QUAY_IO_CREDS')
```

Comment:
same as above

Reply:
Yeah, actually I was wondering the same question. @jduimovich wdyt?

(Regardless of the answer, I don't think it needs to block merging this PR. We could figure it out and make any required changes in later PRs.)

Reply:
Actually I think the intention is to start using `IMAGE_REGISTRY_USER` and `IMAGE_REGISTRY_PASSWORD` instead of QUAY_IO_CREDS, but make sure that QUAY_IO_CREDS_PSW and QUAY_IO_CREDS_USR still work if IMAGE_REGISTRY_USER/PASSWORD are not set. There's some complications with Jenkins though IIUC. Dealing with all that is out of scope for this PR.